packages/perseus/src/widgets/label-image/marker.tsx

```diff
@@ -227,10 +235,20 @@ const styles = StyleSheet.create({
 
         animationDirection: "alternate",
         animationDuration: "0.8s",
-        animationIterationCount: "infinite",
         animationTimingFunction: "ease-in",
 
         transformOrigin: "50% 50%",
+
+        animationIterationCount: "0",
```

Comment:
Is this line necessary when you'll always be providing one of the other two classes?

Reply:
Thanks for clarifying. That makes sense!